--- ./node_modules/react-native/React/Base/RCTModuleMethod.mm
+++ ./node_modules/react-native/React/Base/RCTModuleMethod.mm
@@ -91,6 +91,7 @@ static BOOL RCTParseSelectorPart(const char **input, NSMutableString *selector)
 static BOOL RCTParseUnused(const char **input)
 {
   return RCTReadString(input, "__attribute__((unused))") ||
+  RCTReadString(input, "__attribute__((__unused__))") ||
   RCTReadString(input, "__unused");
 }
